securing_apps/topics/token-exchange/token-exchange.adoc

```diff
@@ -194,11 +193,18 @@ curl -X POST \
     http://localhost:8080{kc_realms_path}/myrealm/protocol/openid-connect/token
 ----
 
-
 The `subject_token` parameter must be an access token for the target realm.  If your `requested_token_type` parameter
 is a refresh token type, then the response will contain both an access token, refresh token, and expiration.  Here's
 an example JSON response you get back from this call.
 
+When the `audience` parameter is not set, the value of the parameter defaults to the client making the token exchange request. If you are passing a `subject_token`, the client to which the token was issued for should either match the client making the request or, if issued to a different client,
+the client making the request should be among the audiences set to the token.
+
+If you are explicitly setting a target `audience`, you should also make sure that the `token-exchange` scope permission is configured for the client set to the `audience` parameter to allow
```

Comment:
Does this imply that I can make an exchange between two clients where none of them have the permissions to perform the token exchange, as long as I don't explicitly set the audience?

Reply:
Not really, if you don't set an audience it is like the client is exchanging a token to itself. The previous statement tries to cover this.

Please, let me know if it needs more clarification.